okhttp/src/test/java/io/grpc/transport/okhttp/OkHttpClientTransportTest.java

```diff
@@ -348,7 +359,7 @@ public void windowUpdateWithInboundFlowControl() throws Exception {
     verify(frameWriter).rstStream(eq(3), eq(ErrorCode.CANCEL));
     listener.waitUntilStreamClosed();
     assertEquals(OkHttpClientTransport.toGrpcStatus(ErrorCode.CANCEL).getCode(),
-        listener.status.getCode());
+                 listener.status.getCode());
```

Comment:
Note that several of us purposefully don't align with the paren. It's fine to leave, but FYI.

That is to say, for lines you are touching it is fine either way, but just "fixing" pre-existing lines isn't likely beneficial as our style simply accepts both ways.